orchestra/tests/helpers/workflow.py

```diff
@@ -825,6 +825,7 @@ def assert_test_dir_v1_loaded(test_case, extra_step=False):
             'name': 'Step 3',
             'description': 'The third step',
             'is_human': True,
+            'completion_ends_project': True,
```

Comment:
do you use this for anything?

Reply:
I use this to test if completion_ends_project got loaded from the workflow version in the test workflow (orchestra/tests/workflows/test_dir/v1/version.json). I have also updated the test workflow to add the  completion_ends_project attribute which is your next comment.